dist/php/businessEmailValidator.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+class BusinessEmailValidator
+{
+    /**
+     * Used to check if provided email address is in list of blocked domains
+     * @access public
+     * @param string $email
+     * The email address we want to check
+     * @return bool
+     * @example 'john@doe.com'
+     */
+    public function isBlocked($email = '')
+    {
+        try {
+
+            // Domains list path
+            $path = 'src/freeEmaislService.json';
```

Comment:
Hey @hajro92 ,
I checked your code and found that the mentioned filename **freeEmaislService.json** is incorrect. So, please update the same.

Reply:
Fixed